docker/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Use the Python Alpine base image
+FROM python:3.12-alpine
+
+# Set the working directory
+WORKDIR /app
```

Comment:
@vasylboliuk please add /app as an ENV variable, not only as a workdir. This way you will be able to reuse it in your Dockerfile